tmt/steps/prepare/install.py

```diff
@@ -582,6 +583,90 @@ def install_debuginfo(self) -> None:
             )
 
 
+class InstallBootc(InstallBase):
+    """Install packages using bootc container image mode"""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize bootc installation for package management"""
+        super().__init__(*args, **kwargs)
+
+        self.engine = self.guest.package_manager.engine
```

Comment:
`self.engine` seems to be rarely used, most of the calls below uses `self.guest.package_manager.engine`.

Reply:
Yes, I removed it and make all calls use `self.guest.package_manager.engine`